### DISCUSSION TITLE
Playtime: Pitch shift (pitch/transpose) control for  audio & midi clips

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#1047] [Ideas]

**Comment by Akognito** [comment]
+

**Comment by Akognito** [comment]
It is very necessary

**Comment by helgoboss** [comment]
Chosen for implementation (issue #1108).

**Comment by helgoboss** [comment]
Pitch shifting will be available in 2.16.2. Both for MIDI and audio. For MIDI, it retriggers already playing notes accordingly, so that it's easier to preview the change with long notes.
